ide/coqide/coqide.ml

```diff
@@ -1514,10 +1517,10 @@ let build_ui () =
         radio "Set diff" 1 ~label:"Show diffs: only _added";
         radio "Set removed diff" 2 ~label:"Show diffs: added and _removed";
       ];
-    item "Show Proof Diffs" ~label:"_Show Proof (with diffs, if set)" ~accel:(modifier_for_display#get ^ "S")
+    item "Show Proof Diffs" ~label:"_Show Proof (with diffs, if set)" ~accel:"<Shift>F2"
```

Comment:
Users tend to find unexplained changes to key bindings annoying.

Reply:
The reason is: Shift-Alt-S is borrowed by “Hide synthesizable types” (not present in CoqIDE 8.16.1).

Reply:
Duplicates should generate an error message at startup.  Far too easy to create them.

Reply:
You mean a developer would create duplicates too easily? GTK probably offers something to detect duplicates, but this would not be my way to work. When I add something to the interface, I tend to start exploring and playing again with everything to check whether it is well integrated and offers a new good global and slightly modified experience. This is relaxing and social. Therefore I’m not in favor of this addition you suggest; in my opinion, the project was not missing such a check, it was missing someone who is enough keen on UX.